mythril/laser/ethereum/instructions.py

```diff
@@ -901,9 +901,7 @@ def balance_(self, global_state: GlobalState) -> List[GlobalState]:
         state = global_state.mstate
         address = state.stack.pop()
 
-        balance = global_state.world_state.balances[
-            global_state.environment.active_account.address
-        ]
+        balance = global_state.world_state.balances[address]
```

Comment:
Bugs like this seem like something that we could easily add tests for, and probably should have had tests for in our opcode tests. :upside_down_face:

Reply:
Yeah